auslib/test/web/test_client.py

```diff
@@ -42,7 +42,8 @@ def setUp(self):
         dbo.setDomainWhitelist(('a.com', 'boring.com'))
         self.client = app.test_client()
         self.view = ClientRequestView()
-        dbo.rules.t.insert().execute(backgroundRate=100, mapping='b', update_type='minor', product='b', data_version=1)
+        dbo.rules.t.insert().execute(backgroundRate=100, mapping='b', update_type='minor', product='b',
+                                     systemCapabilities="SSE2", data_version=1)
```

Comment:
This new rule affects testVersion4Get() in this file, because it gets the new SSE2 rule but then doesn't match and the test fails. You may need a product to keep the tests independent, in which case the whitelist will need changing but you'll need to merge now that the per-product whitelist landed.